test/components/flexjar/FlexjarTest.tsx

```diff
@@ -61,10 +61,10 @@ describe('Flexjar', () => {
     ).to.exist;
   });
 
-  it('renders feedback form content after radiobutton click Ja', () => {
+  it('renders feedback form content after radiobutton click Ja', async () => {
     renderFlexjar();
     const radio = screen.getByRole('radio', { name: 'Ja' });
-    radio.click();
+    fireEvent.click(radio);
```

Comment:
Fikk en drøss med `"Warning: An update to Flexjar inside a test was not wrapped in act(...)."` men ordnet seg ved å bytte til dette. Tror også `fireEvent` / `userEvent` er mer riktig måte å trykke på elementer i react testing library.